integration-tests/jackson/model/pom.xml

```diff
@@ -18,7 +18,7 @@
     <name>Quarkus - Integration Tests - Jackson - model</name>
 
     <properties>
-        <jackson.version>2.9.9.20190807</jackson.version>
+        <jackson.version>2.10.0</jackson.version>
```

Comment:
Can't we get rid of that one? I don't see why it would be a good idea to have that.

Reply:
Done